charts/karpenter/README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
I still see `v1.1.0` listed in the `hugo.yaml` file under the `latest_release_version` -- should we update this to the latest patch version?